File: Heat/Controllers/HomeController.vb
Imports Microsoft.AspNet.Identity.EntityFramework
Imports Microsoft.AspNet.Identity
Imports log4net

Public Class HomeController
    Inherits System.Web.Mvc.Controller

    Private _logger As ILog

    Sub New()
        _logger = LogManager.GetLogger(GetType(HomeController))
    End Sub

    Function Index() As ActionResult
        _logger.Debug("Index function entered")
        Return View()
    End Function

    Function About() As ActionResult
        ViewData("Message") = "Your application description page."

        Return View()
    End Function

    Function Contact() As ActionResult
        ViewData("Message") = "Your contact page."

        Return View()
    End Function

    Function test() As ActionResult
        Return View()
    End Function
End Class

File: Heat.ConvertedToC#/Controllers/HomeController.cs
using System.Web.Mvc;
using log4net;
namespace Heat
{

    public class HomeController : System.Web.Mvc.Controller
	{


		private ILog _logger;
		public HomeController()
		{
			_logger = LogManager.GetLogger(typeof(HomeController));
		}

		public ActionResult Index()
		{
			_logger.Debug("Index function entered");

			return View();
		}

		public ActionResult About()
		{
			ViewData["Message"] = "Your application description page.";

			return View();
		}

		public ActionResult Contact()
		{
			ViewData["Message"] = "Your contact page.";

			return View();
		}

		public ActionResult test()
		{
			return View();
		}
	}
}
